org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/extensions/contentmodel/utils/ReflectionUtils.java

```diff
@@ -0,0 +1,54 @@
+/*******************************************************************************
+* Copyright (c) 2020 Red Hat Inc. and others.
+* All rights reserved. This program and the accompanying materials
+* which accompanies this distribution, and is available at
+* http://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Contributors:
+*     Red Hat Inc. - initial API and implementation
+*******************************************************************************/
+package org.eclipse.lemminx.extensions.contentmodel.utils;

```

Comment:
I would like to avoid doing that to avoid promoting Java Reflection. In Xerces case we need to do that to fix some bugs or improve some features but as some fields are not accessible we need Java Reflection.  In other words, I hope Java Reflection is required only for Xerces.